website/docs/docs/explore/access-dbt-insights.md

```diff
@@ -152,6 +152,15 @@ To access <Constant name="explorer" />, click on the **<Constant name="explorer"
 
 <Lightbox src="/img/docs/dbt-insights/insights-explorer.png" width="90%" title="dbt Insights integrated with dbt Explorer" />
 
+### Set Jinja environment
+
+Set the compilation environment for your Jinja to change how table references are rendered. This feature:
+- Supports "typed" environments marked as `Production`, `Staging`, and/or `Development`.
+- Enables you to point to staging environments for Semantic Layer queries (development environments not supported).
+- Still uses the individual user credentials, so users must have appropriate access to query `PROD` and `STG`. 
+
+<Lightbox src="/img/docs/dbt-insights/insights-jinja-environment.png" width="90%" title="Set the environment for your Jinja context" />
```

Comment:
so sorry, but i think your screenshot got caught before we updated what staging would look like! i can send you another one (or if you have a project that does have staging defined, the pill should be green~

Reply:
Got it!